src/server/game/Entities/Player/Player.cpp

```diff
@@ -26712,3 +26768,237 @@ std::string Player::GetDebugInfo() const
+        return true;
+
+    uint32 ctime = GetLastMoveClientTimestamp();
+    if (ctime)
+    {
+        if (ToUnit()->IsFalling() || IsInFlight())
+            return true;
+
+        if (GetTransport() || GetVehicle() || GetVehicleKit())
+            return true;
+
+        if (HasAuraType(SPELL_AURA_CONTROL_VEHICLE))
+            return true;
+
+        if (HasUnitMovementFlag(MOVEMENTFLAG_ONTRANSPORT))
+            return true;
+
+        if (!IsControlledByPlayer())
+            return true;
+
+        if (GetPlayerMovingMe())
+        {
+            if (GetPlayerMovingMe()->UnderACKmount())
+                return true;
+
+            if (GetPlayerMovingMe()->IsSkipOnePacketForASH())
+            {
+                GetPlayerMovingMe()->SetSkipOnePacketForASH(false);
+                return true;
+            }
+        }
+        else
+            return true;
+
+        Position npos = movementInfo.pos;
+        if (sWorld->getBoolConfig(CONFIG_ANTICHEAT_IGNORE_CONTROL_MOVEMENT_ENABLED))
+        {
+            if (HasUnitState(UNIT_STATE_NOT_MOVE) && !UnderACKRootUpd())
+            {
+                bool unrestricted = npos.GetPositionX() != GetPositionX() || npos.GetPositionY() != GetPositionY();
+                if (unrestricted)
+                {
+                    TC_LOG_INFO("anticheat", "CheckMovementInfo :  Ignore controll Hack detected for Account id : %u, Player %s", GetSession()->GetAccountId(), GetName().c_str());
+                    sWorld->SendGMText(LANG_GM_ANNOUNCE_MOVE_UNDER_CONTROL, GetSession()->GetAccountId(), GetName().c_str());
+                    return false;
+                }
+            }
+        }
+
+        if (HasUnitState(UNIT_STATE_IGNORE_ANTISPEEDHACK))
+            return true;
+
+        float distance, movetime, speed, difftime, normaldistance, delay, delaysentrecieve, x, y;
+        distance = GetExactDist2d(npos);
+
+        if (!jump && !CanFly() && !isSwimming())
+        {
+            float diffz = fabs(movementInfo.pos.GetPositionZ() - GetPositionZ());
+            float tanangle = distance / diffz;
+
+            if (movementInfo.pos.GetPositionZ() > GetPositionZ() &&
+                diffz > 1.87f &&
+                tanangle < 0.57735026919f) // 30 degrees
+            {
+                TC_LOG_INFO("anticheat", "CheckMovementInfo :  Climb-Hack detected for Account id : %u, Player %s, diffZ = %f, distance = %f, angle = %f ", GetSession()->GetAccountId(), GetName().c_str(), diffz, distance, tanangle);
+                sWorld->SendGMText(LANG_GM_ANNOUNCE_WALLCLIMB, GetName().c_str(), diffz, distance, tanangle);
+                return false;
+            }
+        }
+
+        uint32 oldstime = GetLastMoveServerTimestamp();
+        uint32 stime = GameTime::GetGameTimeMS();
+        uint32 ping, realping;
+        movetime = movementInfo.time;
+        realping = GetSession()->GetLatency();
+        ping = realping;
+        if (ping < 60)
```

Comment:
use std::min instead

Reply:
maybe std::max ? =)

Reply:
if ping is less then 60, set to 60. So you want std::min, unless i misunderstood something :P set min value to 60 and other value to whatever realping returns